package.json

```diff
@@ -80,7 +81,7 @@
     "eslint-config-google": "^0.14.0",
     "eslint-plugin-react": "^7.20.3",
     "husky": "^4.2.5",
-    "jest": "24.9.0",
+    "jest": "^26.4.0",
```

Comment:
It is not needed to upgrade jest at all, it may cause a problem within locked dependencies. I think you messed up with the problem you had, but this should remain the same as before